frameworks/SapientML/requirements.txt

```diff
@@ -0,0 +1,3 @@
+sapientml
+openml
+boto3==1.26.98
```

Comment:
Haven't tried it yet, but it looks like the exec file does not depend on these dependencies. What are they for?

Reply:
We have updated the requirements and removed the unwanted dependencies.